specification/adp/resource-manager/readme.cli.md

```diff
@@ -2,11 +2,10 @@
 
 These settings don't need to apply `--cli` on the command line.
 
-``` yaml
-cli:
-  cli-directive:
-    - where:
-        type: DataPoolLocation
-        prop: encryption
-      cli-flatten: true
+``` yaml $(azure-cli-extension-folder) != ''
+directive:
+- from: swagger-document
+  where: $.definitions.DataPoolLocation
+  transform: >
+    $.properties.encryption["x-ms-client-flatten"] = true;
```

Comment:
Can you move this into readme.az.md ? after that, you don't need $(azure-cli-extension-folder) != ''

Reply:
Can I delete readme.cli?

Reply:
Actually no, readme.cli.md is for all command line tools not only including azure-cli